app/email/smtp/EmailSenderProvider.java

```diff
@@ -0,0 +1,40 @@
+package email.smtp;
```

Comment:
Don't think it's worth the effort, but it would be more consistent to move the `email` package into the `demo` package ;-)

Reply:
I see your point, but if `demo` is actually `controllers` then that's the reason why I kept the email settings/providers in their own package as in M6 (not controllers). So I'm just assuming `demo` is actually called `controllers` all the time.